torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -916,7 +916,29 @@ const std::vector<std::string> functions = {
         def gelu(self : Tensor, *, approximate : str):
             result = torch.gelu(self, approximate=approximate)
             def backward(grad_output):
-                return torch.gelu_backward(grad_output, self, approximate=approximate), None
+                M_SQRT2 = 1.41421356237309504880
+                M_SQRT1_2 = 0.70710678118654752440
+                M_2_SQRTPI = 1.12837916709551257390
+                if approximate == "tanh":
```

Comment:
I get nervous whenever I see control flow. In the case for approximate, `torch.nn.modules.GELU` has approximate annotated as `__constant__` so that should be fine.
But using gelu api directly with approximate as an input would stop fusion of gelu to things outside of the block. Fusing gelu_backward with neighboring ops is important.

I know this is harder, but could we instead inject the infinitely differentiable primitive implementation at `gelu_backward` instead? Basically adding an entry here for `gelu_backward` while keeping the entry for `gelu` unchanged.

cc'ing @kevinstephano

Reply:
Let me know if I understand correctly. So I need an entry here for gelu_backward and also define the gelu_double_backward?

I found an implementation of gelu_double_backward from 

https://github.com/pytorch/pytorch/blob/84c8a9f88e15b6d94407e8d1839db1ea8eb6ac6a/torch/csrc/autograd/FunctionsManual.cpp#L3026-L3030

And adding it to `symbolic_script.cpp`: https://github.com/yueyericardo/pytorch/commit/88a6b37e43265e745c95e811ed3f893320ddb529
But I found that 
1. If I call `torch.autograd.grad()` to calculate derivatives: it call `GeluBackwardCUDAKernelImpl` for derivatives instead of the fused kernel.
2. If I run `out.backward()`: it will match to the gelu_backward schema in `torch/csrc/jit/codegen/cuda/parser.cpp`, instead of the entry define at `symbolic_script.cpp`

https://github.com/pytorch/pytorch/blob/8d753c806275fb3436600749aaec87575dd0fabb/torch/csrc/jit/codegen/cuda/parser.cpp#L4095-L4098